_data/tools/hiera.json

```diff
@@ -7,86 +7,120 @@
     {
       "name": "hiera-cloudformation",
       "url": "https://github.com/fanduel/hiera-cloudformation",
+      "name_v5": "",
+      "url_v5": "",
       "description": "AWS CloudFormation backend."
     },
     {
       "name": "hiera-consul",
       "url": "https://github.com/lynxman/hiera-consul",
+      "name_v5": "",
+      "url_v5": "",
       "description": "Consul backend."
     },
     {
       "name": "hiera-etcd",
       "url": "https://github.com/garethr/hiera-etcd",
+      "name_v5": "",
+      "url_v5": "",
       "description": "Etcd distributed configuration store backend."
     },
     {
       "name": "hiera-eyaml",
       "url": "https://github.com/TomPoulton/hiera-eyaml",
+      "name_v5": "",
```

Comment:
It should be `hiera-eyaml` with `url_v5` https://github.com/voxpupuli/hiera-eyaml

Reply:
changed accordingly